specs/1128-EXPT-export_private_key.md

```diff
@@ -0,0 +1,7 @@
+## Export private key
```

Comment:
if we are being picky there is probably a missing (but simple) key details AC. Something like:

'There is a way to export a private key'

Then just whack the link into the key details unit test checking the element exists